### ISSUE TITLE
Add term soft-tissue tumor of the scalp

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10345]
**Preferred term label:**
Soft-tissue tumor of the scalp

**Synonyms**
Soft tumour of the scalp, scalp soft tumor, scalp neoplasm

**Definition (free text, please give PubMed ID)**
A soft-tissue tumor (abnormal growth of tissue) of the scalp
PMID: 23913195

**Parent term (use hpo.jax.org/app)**
Neoplasm of head and neck HP:0012288

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example: soft-tissue sarcoma (National Cancer Institute, https://www.cancer.gov/types/soft-tissue-sarcoma/hp). Other examples: PMID1715206, PMID: 456114

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
I think we would prefer to code the specific type of tumor? What is the use case?

**Comment by azankl** [comment]
@pnrobinson I think it came up in a paper as a descriptive feature, without further specification of the type of tumour,  but I agree, makes more sense if we knew the exact type of tumour. Ok to drop this term request.

**Comment by pnrobinson** [comment]
OK, closing
